### ISSUE TITLE
Enable Roblox

### ISSUE THREAD

**Issue by dawkins666** [nfriedly/node-unblocker#142]

**Comment by nfriedly** [comment]
This thing, right? https://www.roblox.com

I haven't dug in, but it's possibly an issue with websockets, which has a partial fix in #137 - you can use a tweaked version of the nodeunblocker.com code at https://github.com/nfriedly/nodeunblocker.com/tree/websockets that uses the web sockets branch of unblocker. Would you try that out and let me know if it works any better?

**Comment by toxicllyy** [comment]
HOW DO

**Comment by HomieRobloxPlayer** [comment]
i am on a school computer and i want to play roblox

**Comment by SliiCkYT101** [comment]
same

**Comment by HomieRobloxPlayer** [comment]
k

**Comment by HomieRobloxPlayer** [comment]
It started when we were younger, you were mine (my boo)
Now another brother's taken over, but it's still in your eyes (my boo)
Even though we used to argue, it's alright (my boo)
I know we haven't seen each other in a while
But you will always be my boo

**Comment by SliiCkYT101** [comment]
wtf

**Comment by HomieRobloxPlayer** [comment]
https://www.youtube.com/watch?v=MFiajpaw_tI

**Comment by toxicllyy** [comment]
why wont it fucking work ;-;

**Comment by SliiCkYT101** [comment]
what?

\

**Comment by nfriedly** [comment]
Please refrain from cursing or posting unrelated comments. Node Unblocker is something I made about 10 years ago and give away for free. Even when I don't have time to respond to every comment, I do take the time to read them.

There are a lot of newer technologies that didn't exist when I first made this, but I have continued updating it as time permits. If someone wants to try out the websockets branch I mentioned in [my earlier comment](https://github.com/nfriedly/node-unblocker/issues/142#issuecomment-643035617), that was able to make a different game work, and may be sufficient to make this one work. Please try it out and report back here with weather or not it works for this game.

**Comment by HomieRobloxPlayer** [comment]
i did not

**Comment by HomieRobloxPlayer** [comment]
oooooooooooooh you gonnna geeet iiiiiiiiiiiiiiiiiit

**Comment by toxicllyy** [comment]
?

**Comment by 2muchguccii** [comment]
can you fix instagram on your website
[image: Screenshot 2020-11-12 at 2.29.13 PM.png]

**Comment by nfriedly** [comment]
@2muchguccii  It looks like the screenshot didn't come through, so it's hard to tell for sure what you're referring to. Is it related accessing Roblox via the proxy?

**Comment by 2muchguccii** [comment]
no your website node unblocker everything works but insta it just keeps
showing a white page and say somethings wrong can you fix it please

**Comment by nfriedly** [comment]
@2muchguccii That's hard to say. Can you file a new issue at https://github.com/nfriedly/node-unblocker/issues/new and ensure the screenshot gets attached, as well as any relevant URLs?

**Comment by 2muchguccii** [comment]
okay i just did

**Comment by Glitterlxver** [comment]
How do I unblock roblox on the webpage?? :(

**Comment by QuiteAFancyEmerald** [comment]
Hopefully this helps to end this chain of odd comments but Roblox isn't a browser game.

**Comment by nfriedly** [comment]
Ooof. I think I'm just going to close this ticket and say that there is very little chance that Unblocker will ever support Roblox then...

Thank you for the clarification.

**Comment by Glitterlxver** [comment]
yes I know roblox isn't a browser game. Well I can't even download it from
microsoft store and it's ok cause I don't really play roblox anymore even
though sometimes I miss it. I'm ok with that though. thankyou for reading
this.

**Comment by itsyodaddykane** [comment]
> It started when we were younger, you were mine (my boo) Now another brother's taken over, but it's still in your eyes (my boo) Even though we used to argue, it's alright (my boo) I know we haven't seen each other in a while But you will always be my boo

girl
